clients/aws-sdk-bedrock-runtime/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## Unreleased
 
+## v0.1.1
+
+### API Changes
+* New stop reason (`MODEL_CONTEXT_WINDOW_EXCEEDED`) for Converse and ConverseStream.
```

Comment:
nit; we won't be getting these kinds of updates from the normal model releases. We should consider if we want to keep this to using the "official" service team release notes rather than writing our own.

Reply:
The "official" service team release note is `New stop reason for Converse and ConverseStream`. I added the `MODEL_CONTEXT_WINDOW_EXCEEDED` part to provide clarity. I'll update to use the exact service note though 

Reference: https://github.com/boto/botocore/blob/develop/CHANGELOG.rst#L252

Reply:
Addressed in d0f4286493d090c395f0d0270bd03d945dd98d47